Mathlib/Topology/NhdsSet.lean

```diff
@@ -71,6 +71,17 @@ theorem mem_nhdsSet_iff_exists : s ∈ 𝓝ˢ t ↔ ∃ U : Set α, IsOpen U ∧
   rw [← subset_interior_iff_mem_nhdsSet, subset_interior_iff]
 #align mem_nhds_set_iff_exists mem_nhdsSet_iff_exists
 
+/-- A proposition is true on a set neighborhood of `s` if it is true on a larger open set -/
```

Comment:
Perhaps: `A proposition is true on a set neighborhood of `s` if and only if it is true on a larger open set.` ?